hubspot/email.go

```diff
@@ -1,11 +1,21 @@
 package hubspot
 
 type Message struct {
-	To  string   `json:"to"`
-	Cc  []string `json:"cc"`
-	Bcc []string `json:"bcc"`
+	To          string   `json:"to"`
+	From        string   `json:"from,omitempty"`
+	SendID      string   `json:"sendId,omitempty"`
```

Comment:
I know that there was one instance in this project that wasn't following this convention but I believe the general convention at bold is to use underscores for json keys vs camel case.

Reply:
i know what you mean but their api standards is a bit weird. as you can see they use camel case. so if i don't match it i don't think it would work
https://developers.hubspot.com/docs/methods/email/transactional_email/single-send-overview
```
{
    "emailId": 2853049635,
    "message": {
        "to": "example@hubspot.com",
        "sendId": "foobar"
    },
    "contactProperties": [
        {
            "name": "first_name",
            "value": "John"
        }
    ],
    "customProperties": [
        {
            "name": "item_1",
            "value": "something they bought"
        }
    ]
}
```

Reply:
Totally makes sense, I wasn't thinking about having to match a third party, long day, sorry about that.